sklearn/feature_selection/_rfe.py

```diff
@@ -366,6 +396,15 @@ class RFECV(RFE):
         feature count and ``min_features_to_select`` isn't divisible by
         ``step``.
 
+    importance_getter : string or callable, optional (default='auto')
+        If 'auto', uses the feature importance either through a ``coef_``
+        attribute or ``feature_importances_`` attribute of estimator.
+        Also accepts a string that specifies an attribute name/path
+        for extracting feature importance (implemented with `attrgetter`).
+        If `callable`, overrides the default feature importance getter.
+        The callable is passed with the fitted estimator and it should
+        return importance for each feature.
+
```

Comment:
This is out of sync with the version above

Reply:
We haven't changed the callable part now right?
Kindly please give more explanation to help me understand why this is out of sync now?

Reply:
I meant you don't give examples here as above

Reply:
Sorry, I didn't see that. Added it and reordered the example a bit to make it easy for reading.